test/Transition-test.js

```diff
@@ -20,6 +20,10 @@ jasmine.addMatchers({
   }),
 })
 
+function findStatefulInstance(wrapper) {
+  return wrapper.find(Transition.__TestingStatefulComponent).instance()
```

Comment:
`wrapper.find('Transtition')` should be fine and avoids the extra static prop

Reply:
Good idea! I wasn't sure if there weren't two components with the `displayName` "Transition" (some HOC perhaps), but it does work. PR updated